src/components/TaskBoard/TaskBoard.module.scss

```diff
@@ -0,0 +1,41 @@
+.board {
+    display: flex;
+    flex-direction: column;
+    width: 623px;
+    height: 779px;
+    position: absolute;
```

Comment:
you should have VERY solid reasons to use position absolute, please remove it or clarify why do you need it